lib/Data/Events.js

```diff
@@ -71,36 +74,28 @@ export class Events {
      * @param {*} [value1]
      * @param {*} [value2]
      * @param {*} [value3]
+     *
+     * @returns {Events}
      */
     emitToChannel(channel, value, value1, value2, value3) {
-        if (this._buffer) {
-            this._buffer.push(() => this.emitToChannel(channel, value, value1, value2, value3));
-        } else {
-            if (this._channels[channel]) {
-                this._channels[channel].emit(value, value1, value2, value3);
-            }
+        if (this._channels[channel]) {
+            this._channels[channel].emit(value, value1, value2, value3);
```

Comment:
This can be rewritten a bit:

```javascript
emitToChannel(channel, ...values) {
    if (this._channels[channel]) {
        this._channels[channel].emit(...values);
    }

    return this;
}
```